src/ray/raylet/node_manager.cc

```diff
@@ -2414,6 +2418,11 @@ void NodeManager::ResubmitTask(const Task &task, const ObjectID &required_object
   RAY_LOG(DEBUG) << "Attempting to resubmit task "
                  << task.GetTaskSpecification().TaskId();
 
+  if (task.GetTaskSpecification().IsDirectCall()) {
+    TreatTaskAsFailed(task, ErrorType::OBJECT_UNRECONSTRUCTABLE);
```

Comment:
What chain of events would lead to this code being run? I'm guessing it tries to look up the task lease -> cannot find the lease -> tries to re-execute the task? Wouldn't that be impossible since there wouldn't be lineage?

Oh I guess some tasks will randomly have lineage from being submitted for RequestWorkerLease()?

Reply:
Yeah unfortunately lineage is still getting logged for the tasks submitted with the initial worker lease request. I might also disable the lineage stash for direct call tasks, but I think we'll have to keep this while actor creation tasks go through the raylet.